src/TTextEdit.h

```diff
@@ -166,6 +166,8 @@ private slots:
     // Set in constructor for run-time Qt versions less than 5.11 which only
     // supports up to Unicode 8.0:
     bool mUseOldUnicode8;
+    // How many "normal" width "characters" are each tab stop apart:
+    unsigned int mTabStopwidth;
```

Comment:
Please make this a normal `int` - using `unsigned` here goes against C++ Core Guidelines:

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines

> Note: No, using unsigned is not a good way to sidestep the problem of ensuring that a value is nonnegative.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es102-use-signed-types-for-arithmetic

> ES.102: Use signed types for arithmetic